helmfile/charts/rclone-sync/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.15 as download
```

Comment:
We have seen DNS problems when using alpine image compared to Ubuntu image. I have not found any definitive answer that it is fixed yet. So i would suggest to switch to a Ubuntu image.

Reply:
Sure I can change that.